examples/query-subscription-consumer/1/query-subscription-consumer

```diff
@@ -0,0 +1,19 @@
+{
+  "version": 3,
+  "payload": {
+    "subscription_id": "1234", 
+    "result": {
+      "data": [
+        {
+          "hello": 50,
```

Comment:
Is this real data? This does not seem like anything that should be published on this topic. Isn't there supposed to be a "count" here?